examples/connect/webhook_ngrok.py

```diff
@@ -68,7 +68,7 @@
 import threading
 import traceback
 
-api = ConnectAPI()
+api = ConnectAPI(dict(autostart_notification_thread=False))
```

Comment:
I can't find where the behaviour is documented in the interface; `ConnectAPI.__init__` would seem to be the logical place.

I think it is also worth writing a comment in the example that `autostart_notification_thread=False` is required to use a webhook.